test/build2file

```diff
@@ -39,6 +37,9 @@ for test_cpp : $tests_sources
 if($cxx.id != 'msvc')
 {
     # Enable all warnings:
+    # cja my clang 15.0 pre-release on Windows doesn't like -pedantic-errors...
+    # further, getting warning spam from -Wgnu-line-marker, not sure why.
+    # clang windows builds remain off in package manifest
```

Comment:
In the `dev` branch I re-activated it for testing, but it only revealed the incompatibility, as reported there: https://github.com/ericniebler/range-v3/issues/1721